README.md

```diff
@@ -1,24 +1,10 @@
 # eslint-plugin-local-rules
 
-A plugin for ESLint that allows you to use project-specific rules, similar to the [`--rulesdir`](http://eslint.org/docs/user-guide/command-line-interface#--rulesdir) command line option ([more](http://eslint.org/docs/developer-guide/working-with-rules#runtime-rules)).
-
-Workaround for https://github.com/eslint/eslint/issues/8769 (previously https://github.com/eslint/eslint/issues/2715).
-
-## Other solutions
-
-- https://github.com/taskworld/eslint-plugin-local
-  - Allows specifying additonal plugin config such as 
-    [`processors`](https://eslint.org/docs/developer-guide/working-with-plugins#processors-in-plugins)
-- https://github.com/not-an-aardvark/eslint-plugin-rulesdir
-  - Allows for a custom rules directory name
-
-## Dependencies
-
-- Requires ESLint version 0.8.0 or higher
```

Comment:
ESLint is now at 9. It's safe to assume everyone is using a version much more recent than 0.8.0